src/styles/containerComposeStyles/index.native.ts

```diff
@@ -1,6 +1,5 @@
-import styles from '@styles/styles';
-import ContainerComposeStyles from './types';
+import Styles from '@styles/styles';
 
-const containerComposeStyles: ContainerComposeStyles = [styles.textInputComposeSpacing];
+const containerComposeStyles = (styles: typeof Styles) => [styles.textInputComposeSpacing];
```

Comment:
same with the type

Reply:
removed it as TS infers it in a much better fashion. Original was just this, pretty blunt IMO:
```typescript
type ContainerComposeStyles = ViewStyle[];
```

Reply:
If you decided to stop using `ContainerComposeStyles`, you should remove `types.ts` file as well.